test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -4608,6 +4624,13 @@
       "path": "system.objectmodel/4.0.12",
       "hashPath": "system.objectmodel.4.0.12.nupkg.sha512"
     },
+    "System.Private.Uri/4.3.2": {
```

Comment:
System.Private.Uri is already listed as `minorMatchOrLower`... but it's not clear what version we were previously referencing. It appears that we weren't referencing?

@fabiocav -- any ideas here? Should we move this to `private` if we weren't referencing it previously?

Reply:
Followed up with @brettsam offline. This is fine as-is, as it's already a dependency of the host (and a framework reference), but now being overridden with the package reference.